index.html

```diff
@@ -609,6 +609,77 @@ <h3>Ed25519Signature2018</h3>
           </pre>
       </section>
 
+      <section id="JsonWebSignature2020">
+
+        <h3>JsonWebSignature2020</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and JWS to perform the digital signature.
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>Signature</dd>
+            <dt>Expected properties</dt>
+            <dd>verificationMethod, jws</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/rdf#URDNA2015">
+https://w3id.org/rdf#URDNA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="https://tools.ietf.org/html/rfc8037#section-3.1">
+  https://tools.ietf.org/html/rfc8037#section-3.1</a>
```

Comment:
Is the intent to default the same as Ed25519Signature2018 above?  (I'm not entirely clear how this suite works.)

Reply:
This suite is functionally identical to both `Ed25519Signature2018` and `EcdsaSecp256k1Signature2019`... both generate create Verity Data and then use a detached jws with special header values to sign it, and both use `alg` `EdDSA` and `ES256K` respectively... the difference is that this suite also supports all the NIST curves (which some of use are forced to use).... and it relies on this table here for providing guidance on other JOSE suites: https://github.com/w3c-ccg/lds-jws2020#supported-jose-algorithms